assets/src/js/batch.jsx

```diff
@@ -140,6 +142,11 @@ var App = React.createClass( {
 		this.toggleProcessing( true );
 	},
 
+	toggleResetModal : function( active ) {
+		if( active === false || active === true ) {
```

Comment:
I don't remember if this project has `eslint` config but this should be `if (` per WP coding standards https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#examples-of-good-spacing